Source/JavaScriptCore/runtime/TemporalObject.cpp

```diff
@@ -600,55 +627,78 @@ double roundNumberToIncrement(double x, double increment, RoundingMode mode)
 
 // RoundNumberToIncrementAsIfPositive ( x, increment, roundingMode )
 // https://tc39.es/proposal-temporal/#sec-temporal-roundnumbertoincrementasifpositive
-Int128 roundNumberToIncrement(Int128 x, Int128 increment, RoundingMode mode)
+Int128 roundNumberToIncrementAsIfPositive(Int128 x, Int128 increment, RoundingMode roundingMode)
 {
-    ASSERT(increment);
-
-    if (increment == 1)
-        return x;
-
+// The following code follows the polyfill rather than the spec, because we don't have float128.
```

Comment:
I would suggest rephrasing this comment and the one below not to refer to "the polyfill", it's not clear from context here what is meant.

Reply:
I added links to the relevant code in the polyfill in [86a28f8](https://github.com/WebKit/WebKit/pull/45603/commits/86a28f87d22a6bd6f24f140318c2fd07be727fdd) -- hopefully that clarifies?